cache/api.github.com/webhook-events-and-payloads.html

```diff
@@ -4948,23 +4931,6 @@ <h2 id="installation_repositories">
           >GitHub App installation</a
         >" REST API.
       </p>
-      <div
-        class="
-          extended-markdown
-          note
-          border
-          rounded-1
-          mb-4
-          p-3
-          color-border-info color-bg-info
-          f5
-        "
-      >
-        <p>
-          <strong>Note:</strong> This event replaces the deprecated
-          <code>integration_installation_repositories</code> event.
-        </p>
-      </div>
```

Comment:
we do have examples for `integration_installation` and `integration_installation_repositories`. I suggest we remove any reference to these deprecated events from this repository, as they are no longer mentioned in the official docs at https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads

Reply:
ping @wolfy1339 what do you think?

Reply:
Sounds good
We already removed them for `api.github.com`, only remaining references are GHES and GitHub AE

Reply:
done via 11fc6a0